lite/pom.xml

```diff
@@ -550,6 +550,11 @@
                         <include>fix_throws_javadoc.sh</include>
                         <!-- all source files in a directory named util, in case the samples start picking those up -->
                         <include>src/**/util/**/*</include>
+                        <!-- Maven Wrapper files -->
+                        <include>.mvn/wrapper/maven-wrapper.jar</include>
```

Comment:
I'd recommend using a wildcard for this, like ``.mvn/wrapper/**/*``, (sort of like line 552), since the ``.mvn/wrapper`` directory operates as a unit, and could have more things in it later. Not critical, but it would be defensive.